set_RMQ_EC2.py

```diff
@@ -0,0 +1,69 @@
+##Should run on local machine
+## And ssh into your AWS accounts then start rabbitmq-server
+
+import paramiko
+
+hostnames = [
+	'ec2-54-173-2-90.compute-1.amazonaws.com'		##Your instance's public name
+	
+]
+## since gonna be same for each host
+k = paramiko.RSAKey.from_private_key_file("/home/aniket/Downloads/rabbitmqkey.pem")
```

Comment:
`"/home/aniket/Downloads/rabbitmqkey.pem"`
things like these can be set as variables and then variables can be used in the code later. 

So later if I want to change the values of these params, I dont have to read the full code, just change the value of the params